engine/src/main/java/org/terasology/monitoring/gui/ChunkMonitorDisplay.java

```diff
@@ -105,13 +104,18 @@ public ChunkMonitorDisplay(int refreshInterval, int chunkSize) {
         addMouseListener(ml);
         addMouseMotionListener(ml);
         addMouseWheelListener(ml);
+        this.followPlayer = true;
```

Comment:
With the removal of `setFollowPlayer` this var itself might be obsolete since there's no way to change it. Probably either keep the setter in expectation that it'll be added as a toggle in the UI sometime or just remove it entirely :-)

Reply:
I decided to keep the setter, so we can open an issue to add that new UI functionality.